libs/output-mapping/tests/Needs/TestSatisfyer.php

```diff
@@ -156,18 +178,29 @@ public static function satisfyTestNeeds(
             self::ensureRemoveBucket($clientWrapper, $removeBucket->getArguments()[0]);
         }
 
+        $testResourceName = substr(
+            sprintf(
+                '%s_%s%s',
+                $reflection->getShortName(),
+                $methodName,
+                $dataName ? '_' . preg_replace('/[^a-zA-Z0-9-_]/', '-', $dataName) : '',
+            ),
+            0,
+            96,
+        );
```

Comment:
Doteď to byl jen název testu.  Kvůli paratestu se musela udělat úprava. aby se tam přidal i název třídy (některé testCases mají stejné názvy testů) + tam je i dataName z dataProvideru.